src/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,133 @@
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+import type { StoryObj, Meta } from '@storybook/react';
+
+import React from 'react';
+import { Badge } from './Badge';
+
+export default {
+  title: 'Molecules/Messaging/Badge',
+  component: Badge,
+  subcomponents: {
+    'Badge.Dot': Badge.Dot,
+    'Badge.Icon': Badge.Icon,
+    'Badge.Text': Badge.Text,
+  },
+  parameters: {
+    layout: 'centered',
+    badges: [BADGE.BETA],
+  },
+} as Meta<Args>;
+
+type Args = React.ComponentProps<typeof Badge>;
+
+export const Dot: StoryObj<Args> = {
+  args: {
+    children: (
+      <>
+        <div
+          aria-label="Ava has notification"
+          className="fpo flex items-center justify-center h-8 w-8"
+        >
+          Ava
+        </div>
+        <Badge.Dot />
+      </>
+    ),
+  },
+};
+
+export const EmptyText: StoryObj<Args> = {
+  args: {
+    children: (
+      <>
+        <div
+          aria-label="Ava has notification"
+          className="fpo flex items-center justify-center h-8 w-8"
+        >
+          Ava
+        </div>
+        <Badge.Text>{''}</Badge.Text>
+      </>
+    ),
+  },
+};
```

Comment:
Still recommended?

Reply:
Nah, will remove story, thanks for the call out